src/Api/Statistics/CollectionStatsLogger.php

```diff
@@ -0,0 +1,138 @@
+<?php
+/**
+ * @file
+ * Collection Statitics Logger
+ */
+
+namespace App\Api\Statistics;
+
+use App\Service\MetricsService;
+use App\Service\StatsLoggingService;
+use Symfony\Component\HttpFoundation\RequestStack;
+
+/**
+ * Class CollectionStatsLogger.
+ */
+class CollectionStatsLogger
+{
+    private $statsLoggingService;
+    private $metricsService;
+    private $requestStack;
+
+    private const SERVICE = 'CoverCollectionDataProvider';
+    private const CLIENT_ID = 'REST_API';
```

Comment:
The `CLIENT_ID` should be extracted from the user authentication token. Yes I known that it may have been in the other PR :-) Just need to remember this...